support-frontend/devrun.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+yarn devrun &
+yarn storybook &
+cd ..; sbt "project support-frontend" devrun
```

Comment:
I'm slightly dubious of the `cd ..` statement, but I'm no bash expert.

Reply:
it's intentional (sbt needs to run from the root) but i'm not sure if it can be improved)